assets/airports/kmia.json

```diff
@@ -167,245 +180,602 @@
-                "JORAY": [["JORAY", "S280"], "OSOGY", "AMZAR", ["STAPL", "A180"], ["MILSY", "A160|S280"], "BOYUR", "HILEY"]
+                "JORAY": ["JORAY", "OSOGY", "AMZAR", "STAPL", "MILSY", "BOYUR"],
+                "OMN": ["OMN", "MALET", "PCMAN", "MLB", "TRV", "PHORD", "SMUGS", "MORGA", "PBI", "BTOGA"],
+                "ZFP": ["ZFP", "STAPL", "MILSY", "BOYUR"]
             },
             "body": ["HILEY", "KAINS", "CIMBA", "JESSS", "RUBOE"],
-            "draw": [["OMN*","MALET","PCMAN","MLB","TRV","PHORD","SMUGS","MORGA","PBI","BTOGA","HILEY"], ["JORAY*","OSOGY","AMZAR","STAPL","MILSY","BOYUR","HILEY"], ["ZFP*","STAPL"], ["HILEY","KAINS","CIMBA","JESSS","RUBOE"]]
+            "rwy": {
+                "8L": [],
+                "8R": [],
+                "9": [],
+                "12": [],
+                "26L": [],
+                "26R": [],
+                "27": [],
+                "30": []
+            },
+            "draw": [
+                ["JORAY*", "OSOGY", "AMZAR", "STAPL", "MILSY", "BOYUR", "HILEY", "KAINS", "CIMBA", "JESSS", "RUBOE"],
+                ["OMN*", "MALET", "PCMAN", "MLB", "TRV", "PHORD", "SMUGS", "MORGA", "PBI", "BTOGA", "HILEY"],
+                ["ZFP*", "STAPL"]
+            ]
         },
         "SSCOT3": {
             "icao": "SSCOT3",
             "name": "Scott Three",
             "entryPoints": {
-                "BAGGS": [["BAGGS", "S280"], "BAARY"],
-                "SZW": ["SZW", "HEVVN", "LEGGT", "PLYER", "SEATE", "JUULI", "BAARY"],
-                "OTK": ["OTK", "HITTR", "TUCKE", "JUULI", "BAARY"],
-                "TAY": ["TAY", "HITTR", "TUCKE", "JUULI", "BAARY"]
+                "BAGGS": ["BAGGS"],
+                "OTK": ["OTK", "HITTR", "TUCKE", "JUULI"],
+                "SZW": ["SZW", "HEVVN", "LEGGT", "PLYER", "SEATE", "JUULI"],
+                "TAY": ["TAY", "HITTR", "TUCKE", "JUULI"]
+            },
+            "body": ["BAARY", "CYY", "DEEDS", "SSCOT", "RUBOE", "WALIP", "LECIT"],
+            "rwy": {
+                "8L": [],
+                "8R": [],
+                "9": [],
+                "12": [],
+                "26L": [],
+                "26R": [],
+                "27": [],
+                "30": []
             },
-            "body": ["BAARY", "CYY", ["DEEDS", "A160|S280"], ["SSCOT", "A100"], "RUBOE", "WALIP"],
-            "draw": [["BAGGS*","BAARY"], ["SZW*","HEVVN","LEGGT","PLYER","SEATE","JUULI","BAARY"], ["OTK*","HITTR","TUCKE","JUULI","BAARY"], ["TAY*","HITTR","TUCKE","JUULI","BAARY"], ["BAARY","CYY","DEEDS","SSCOT","RUBOE","WALIP"]]
+            "draw": [
+                ["BAGGS*", "BAARY", "CYY", "DEEDS", "SSCOT", "RUBOE", "WALIP", "LECIT"],
+                ["OTK*", "HITTR", "TUCKE", "JUULI", "BAARY"],
+                ["SZW", "HEVVN", "LEGGT", "PLYER", "SEATE", "JUULI"],
+                ["TAY*", "HITTR"]
+            ]
         }
     },
     "spawnPatterns": [
```

Comment:
I'd imagine WINCO2 isn't the only departure used

Reply:
for sure 😉 this was fixed.